README.Rmd

```diff
@@ -17,13 +17,33 @@ knitr::opts_chunk$set(
+EpiNow2 estimates the time-varying reproduction number, growth rate, and doubling time using a range of open-source tools ([Abbott et al.](https://doi.org/10.12688/wellcomeopenres.16006.1)), and current best practices ([Gostic et al.](https://doi.org/10.1371/journal.pcbi.1008409)). It aims to help users avoid some of the limitations of naive implementations in a framework that is informed by community feedback and is actively supported.
+
+Forecasting is also supported for the time-varying reproduction number, infections, and reported cases using the same generative process approach as used for estimation.
+
+<details> <summary> More details </summary>
+
+EpiNow2 estimates the time-varying reproduction number on cases by date of infection (using a similar approach to that implemented in [`{EpiEstim}`](https://github.com/mrc-ide/EpiEstim)). Imputed infections are then mapped to observed data (for example cases by date of report) via a series of uncertain delay distributions (in the examples in the package documentation these are an incubation period and a reporting delay) and a reporting model that can include weekly periodicity. 
 
 Uncertainty is propagated from all inputs into the final parameter estimates, helping to mitigate spurious findings. This is handled internally. The time-varying reproduction estimates and the uncertain generation time also give time-varying estimates of the rate of growth.
 
-The default model uses a non-stationary Gaussian process to estimate the time-varying reproduction number and then infer infections. Other options include:
+</details>
+
+<details> <summary> Models provided </summary>
+
+`{EpiNow2}` is designed to be used via a single function call to two functions:
+
+* [`epinow()`](https://epiforecasts.io/EpiNow2/reference/epinow.html): Estimate Rt and cases by date of infection and forecast these infections into the future.
+
+* [`regional_epinow()`](https://epiforecasts.io/EpiNow2/reference/regional_epinow.html): Efficiently run `epinow()` across multiple regions in an efficient manner.
+
+These two functions call [`estimate_infections()`](https://epiforecasts.io/EpiNow2/reference/estimate_infections.html), which works to reconstruct cases by date of infection from reported cases.
+
+[`estimate_infections()`](https://epiforecasts.io/EpiNow2/reference/estimate_infections.html) can be used on its own to infer the underlying infection case curve from reported cases and estimate Rt. Estimating the underlying infection case curve via back-calculation (and then calculating Rt) is substantially less computationally demanding than generating using default settings but may result in less reliable estimates of Rt.
```

Comment:
I think no I would consider leading with `estimate_infections` and then introducing `epinow` and `regional_epinow` within that section vs doing it in this order